vite.config.ts

```diff
@@ -18,11 +18,16 @@ export default defineConfig({
       },
     },
     rollupOptions: {
-      external: ['react', 'next/cache'],
+      external: ['react', 'react/jsx-runtime', 'next/cache'],
```

Comment:
@edodusi We were bundling all this before to the final bundle?

Reply:
I removed this, it was a leftover for a problem I had previously